src/internal/utils/sort.ts

```diff
@@ -103,11 +103,15 @@ export function getDefaultPivotTableSort(afm: AFM.IAfm): AFM.SortItem[] {
     return [];
 }
 
-function getDefaultBarChartSort(afm: AFM.IAfm, resultSpec: AFM.IResultSpec): AFM.SortItem[] {
+function getDefaultBarChartSort(
+    afm: AFM.IAfm,
+    resultSpec: AFM.IResultSpec,
+    isSimpleStackMeasures: boolean = false,
+): AFM.SortItem[] {
     const viewByAttributeIdentifier = getFirstAttributeIdentifier(resultSpec, VIEW_BY_DIMENSION);
     const stackByAttributeIdentifier = getFirstAttributeIdentifier(resultSpec, STACK_BY_DIMENSION);
 
-    if (viewByAttributeIdentifier && stackByAttributeIdentifier) {
+    if ((viewByAttributeIdentifier && stackByAttributeIdentifier) || isSimpleStackMeasures) {
         return [getAttributeSortItem(viewByAttributeIdentifier, SORT_DIR_DESC, true)];
```

Comment:
Can we sort `ascending` at SDK user level?

Reply:
yes

Reply:
How do we do that?
I see we hardcode `SORT_DIR_DESC`.

Reply:
this is the src/internal folder. it handle for pluggable chartType. Then hardcode `SORT_DIR_DESC` mean that set default sort for stacked bar chart is `desc`